src/vendors/autocomplete.js

```diff
@@ -91,7 +80,7 @@ export default function autoComplete(options) {
             that.sourcePending = o.source(val);
             that.sourcePending.then(source => {
               that.sourcePending = null;
-              if (source !== null) {
+              if (source !== null && document.activeElement === that) {
                 suggest(source);
```

Comment:
Here is the delicate change.  
Do you see any situation / platform where it would be expected to show the suggest list while the input field would not be the active element ?

Reply:
Actually, this is something the design asked to do in future iterations on mobile… The virtual keyboard takes a lot of screen space when the field is focused, so they proposed to be able to scroll the list in full screen… We wanted to finish the component refacto first, before looking for an elegant solution to do that.

Reply:
In any case, I think we'll implement that differently, as this autocomplete lib is supposed to disappear entirely. So any clean-up that can happen before is welcome :)

Reply:
a similar fix is on line line 77 for firefox on android. I'm not aware of any situation where it should be an issue